lisa/node.py

```diff
@@ -514,3 +514,24 @@ def from_requirement(
         self._list.append(node)
 
         return node
+
+
+def quick_connect(runbook: schema.Node, name: str = "", index: int = -1) -> Node:
+    """
+    setup node information and initialize conneciton.
+    """
+    node = Node.create(index, runbook, logger_name=name)
+    if isinstance(node, RemoteNode):
+        node.set_connection_info_by_runbook()
+    node.initialize()
+
+    return node
+
+
+def quick_connect_remote(
+    runbook: schema.RemoteNode, name: str = "", index: int = -1
```

Comment:
You'd better repeat what the inner method states here, so to avoid people having to follow source tree just to understand what a function does

Reply:
After reviewed the logic, the method is removed. The most places should use Node, not RemoteNode.